### PR TITLE
Feat: refreshed resources

### LINKED ISSUE #269: Replace Resources on calitp.org with refreshed versions for improved accessibility
I have uploaded a folder [here](https://drive.google.com/drive/folders/1ZcWc_PIKt9rCO5aZ-Zyswfkw9r4DKmMl?usp=sharing) with updated versions of the resources currently hosted on calitp.org/resources. 

We would like to replace the existing resources with the refreshed versions which contain a number of accessibility improvements, mostly the addition of tags and cleanup of logo tags.

### PR DESCRIPTION
**Pull request by angela-tran**
Closes #269

### CODE REVIEW

**Review by thekaveman — CHANGES_REQUESTED** [review]
Let's go ahead and clean up all the file names, since we are changing the links.

Absolutely no good reason to have spaces in a filename on a website.

**Comment by angela-tran** [comment]
@mrose914 I'm going to clean up the names of the PDF files being added here so that spaces are replaced with `.` and any files specifically from Cal-ITP are prefixed with `Cal-ITP`. Does that sound ok to you?

File: src/_resources/one-pager-data-playbook-making-your-gtfs-publicly-available.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 date: "2023-07-02T17:00:00-07:00"
 title: "Data Playbook: Making your GTFS publicly available"
-asset: https://drive.google.com/file/d/1G8hp5PwaR9xUdiLkq_5tXpYWqle7qwwj/view
+asset: "Cal-ITP.Data.Playbook.1-Making.Your.GTFS.Publicly.Available.pdf"
```
Comment (thekaveman) [inline src/_resources/one-pager-data-playbook-making-your-gtfs-publicly-available.md:4]: Let's remove this weird `.1`

**Comment by angela-tran** [comment]
Preview site is updated with cleaned-up file names: https://deploy-preview-273--cal-itp-website.netlify.app/resources

**Review by thekaveman — CHANGES_REQUESTED** [review]

File: src/_resources/one-pager-data-playbook-making-your-gtfs-publicly-available.md (hunk 1 above)
Comment (angela-tran) [reply src/_resources/one-pager-data-playbook-making-your-gtfs-publicly-available.md]: Done in https://github.com/cal-itp/calitp.org/pull/273/commits/c40fc935edb7a3fadcb383f482c49285a00607f7

**Review by thekaveman — APPROVED** [review]

**Comment by angela-tran** [comment]
I'll go ahead and merge this today

**Pull request merged**